.github/workflows/autobump.yml

```diff
@@ -139,6 +139,7 @@ env:
     cloud-nuke
     cloudflare-wrangler
     cloudformation-guard
+    clusterawsadm
```

Comment:
no need to add it here as of now, it would trigger tsc review

Reply:
Done